### DISCUSSION TITLE
Continuation of Sigstore discussion from 10/29 Touchpoint

### DISCUSSION THREAD

**Discussion by byjrack** [todogroup/ospology#32] [OSPO Discussions]
https://www.sigstore.dev/how-it-works

I couldn't do it justice so won't try, but 💯that like Let's Encrypt making it trivial for both the producer and consumer to leverage the service will be key to adoption.

@anajsana or @caniszczyk do you have thoughts on how we could pull in someone from the sigstore project to one of our meetings to get their ideas on how their engagement strategy will play out?  Before we start giving Github a bunch of ideas on what might be good having a better alignment to the core project may allow us to keep those Hub asks precise.

**Comment by caniszczyk** [comment]
I think what we can do is setup a "Security Supply Chains and the Role of
OSPOs" ospology session by the SigStore folks.

We can invite some folks there to speak and we can get some concrete items
out of that meeting for GH/GL etc

I'll work with Ana to get something on the calendar for early next year

On Fri, Oct 29, 2021 at 12:07 PM Justin Rackliffe ***@***.***>
wrote:

> https://www.sigstore.dev/how-it-works
>
> I couldn't do it justice so won't try, but 💯that like Let's Encrypt
> making it trivial for both the producer and consumer to leverage the
> service will be key to adoption.
>
> @anajsana <https://github.com/anajsana> or @caniszczyk
> <https://github.com/caniszczyk> do you have thoughts on how we could pull
> in someone from the sigstore project to one of our meetings to get their
> ideas on how their engagement strategy will play out? Before we start
> giving Github a bunch of ideas on what might be good having a better
> alignment to the core project may allow us to keep those Hub asks precise.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/todogroup/ospology/discussions/32>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAAPSIJES2KNTE4HVDMHPELUJLBC3ANCNFSM5G7VKDWA>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
>


-- 
Cheers,

Chris Aniszczyk
https://aniszczyk.org

**Reply by byjrack** [reply]
Yup I believe cosign is more focused on container images so knowing if they have a vision toward enabling other artifacts in the same framework and how we can help as OSPOs support that.
